actors/serializers.py

```diff
@@ -0,0 +1,8 @@
+from rest_framework import serializers
+from . import models
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+    	model = models.Actor
+    	fields = '__all__'
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `all.whitespace`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp3p5cz0y5/actors/serializers.py
+++ b/tmp/tmp3p5cz0y5/actors/serializers.py
@@ -5,4 +5,4 @@
 class ActorSerializer(serializers.ModelSerializer):
     class Meta:
     	model = models.Actor
-    	fields = '__all__'
+        fields = '__all__'
```